node/src/actors/chain_manager/mod.rs

```diff
@@ -2171,20 +2171,23 @@ impl ChainManager {
 
     /// Return the value of the version field for a block in this epoch
     fn tapi_signals_mask(&self, epoch: Epoch) -> u32 {
-        let Tapi {
-            oppose_wip0014,
-            oppose_wip0016,
-        } = &self.tapi;
+        let Tapi { oppose_wip0017 } = &self.tapi;
 
         let mut v = 0;
-        if !oppose_wip0014
-            && !oppose_wip0016
+        // Bit 1
+        let bit = 1;
+        if !oppose_wip0017
+            && !self
+                .chain_state
+                .tapi_engine
+                .wip_activation
+                .contains_key("WIP0017")
```

Comment:
This condition is new: it means that miners will stop signaling when the WIP is activated. Not sure if this is the desired behavior, because this field is also used as a census to see the percentage of miners that are on the latest version. Also, bit 0 is set to 0 in this PR while currently in mainnet it is set to 1. That doesn't break anything consensus-wise but it may be unexpected.

Reply:
In my opinion, this bit only should be used to signaling, and after achieve the necessary votes, it is not longer required. We should remove the activation of this bit, to avoid overlap bit problems in the future

Reply:
I have mixed opinions. WIP-0014 does not define what to do because it is consensus neutral. It is true that setting bit position 0 to value 0 at this point may come as a surprise, but also we know for certain that we'll need to do so at some point in the coming years. So maybe it is time to define whether we are OK with setting them to 0 upon next bit position being assigned, or we make a commitment to set all bits to 0 when we assign bit 31 (although that removes the ability of signaling 30 and 31 at the same time).

Reply:
I think being able to query which (ARS) nodes have signalled support for a certain WIP after the WIP is passed or defeated is quite a useful feature. Yes, in the (distant) future, bits will be reused and at that point this'll become a problem, but maybe that is not something that should be solved right now.

Reply:
Could be an option to leave as before and open an issue with a discussion about that

Reply:
Yes, let's just leave it like it was before.